webflux6-micrometer/src/main/resources/application.yaml

```diff
@@ -0,0 +1,42 @@
+management:
+  # only expose /health from actuator
+  endpoints:
+    enabled-by-default: false
+    web:
+      base-path: ''
+      path-mapping:
+        health: /health
+  endpoint:
+    health:
+      enabled: true
+
+  tracing:
+    brave:
+      # Note: There is no property to bind ${brave.traceId128Bit:false}
+      span-joining-supported: ${brave.supportsJoin:true}
+    sampling:
+      # There doesn't seem to be a way to skip tracing /health by property
+      probability: 1.0
+    # Propagates a field named 'user_name' downstream
+    # Note: In Spring Boot 3 it is not yet possible to map 'user_name' to the
+    # correlation field 'userName'.
+    baggage:
+      remote-fields:
+        - user_name
+      correlation:
+        fields:
+          - user_name
+    propagation:
+      # Use b3 to mix with other examples without adding brave-propagation-w3c
+      type: b3_multi
+  zipkin:
+    tracing:
+      # Note: There is no property to bind ${brave.localServiceName:${spring.application.name}}
+      endpoint: ${zipkin.baseUrl:http://127.0.0.1:9411}/api/v2/spans
+
+logging:
+  level:
+    root: INFO
+  pattern:
+    # Note: Logs don't appear to have any correlation fields assigned.
```

Comment:
I hope log correlation isn't a w3c only thing.. I perused a bunch of github issues and seems people have been steered towards w3c, but I don't know if that's just because maintainers like it more, or if logic hasn't been stitched together to make normal brave work like before (even spring 2.5 can do this). @shakuzen if you know the scoop lemme know!